onnxruntime/python/tools/tensorrt/perf/perf.sh

```diff
@@ -1,4 +1,41 @@
 #!/bin/bash
 
-python3 benchmark.py -r validate
-python3 benchmark.py -r benchmark -i random -t 100
+SYMBOLIC_SHAPE_INFER="/home/hcsuser/repos/onnxruntime/onnxruntime/python/tools/symbolic_shape_infer.py"
```

Comment:
same comment as ort_build_latest.py ,
looks like this script assumes a specific filesystem layout/user account is present on the system (hcsuser)

Reply:
Same modification as ort_build_latest.py